src/frontend/components/DashboardVideoPane/index.spec.tsx

```diff
@@ -282,4 +282,17 @@ describe('<DashboardVideoPane />', () => {
       cleanup();
     }
   });
+
```

Comment:
You have to update your expectation about `getText` in `renders & starts polling for the video` test

```diff
- Your video is currently processing. This may take up to an hour. Please come back later.

+ Your video is currently processing. This may take up to an hour. You can close the window and come back later.
```